cleanup_utils/aws_cleanup.py

```diff
@@ -183,10 +199,7 @@ def run_cleanup(self):
             )
             # If the user has any old keys, proceed with deletion
             if old_keys:
-                user_policy = self.iam_client.get_user_policy(
-                    UserName=user['UserName'],
-                    PolicyName=DEFAULT_POLICY_NAME
-                )
+                user_policy = self.get_user_policy(user['UserName'])
                 # If user policy exists
                 if user_policy:
```

Comment:
@giovannicimolin, should some thing be logged when `user_policy` is `None`?

Reply:
Yes, and even when the user doesn't have that policy, but has `integration` on the name, it should be deleted.
I'll implement this behaviour.